app/assets/images/payment_icons/clave_telered.svg

```diff
@@ -0,0 +1 @@
-.68-1.55-1.51V3.09c0-.84.69-1.51 1.55-1.51h30.75c.86 0 1.55.67 1.56 1.51v17.82c0 .83-.7 1.51-1.56 1.51v.03"/><path id="pi-clave_telered-path18" fill="#ee2926" d="M33.91 21.95c.85 0 1.54-.66 1.54-1.48V3.48c0-.82-.69-1.49-1.54-1.49H4.23c-.84 0-1.53.66-1.53 1.49v16.99c0 .81.69 1.48 1.53 1.48h29.68"/><path id="pi-clave_telered-path20" fill="#121815" d="M16.19 10.24h19.22V6.19H16.19v4.05"/><path id="pi-clave_telered-path22" fill="#fff" d="M11.47 15.84l-.28 1.07c-.78.38-1.7.53-2.57.53-2.79 0-4.89-1.64-4.89-4.21 0-3.27 2.35-4.76 5.18-4.76.87 0 1.89.21 2.65.61-.12.44-.19.87-.29 1.3h-.19c-.48-.67-1.3-1.15-2.15-1.15-2.16 0-3.03 1.99-3.03 3.83 0 2.14 1.09 3.65 3.09 3.65.8 0 1.74-.4 2.33-.94l.17.07"/><path id="pi-clave_telered-path24" fill="#fff" d="M14.76 17.29c-.3-.02-1.5-.02-1.8 0V7.77c.3.03 1.5.03 1.8 0v9.52"/><path id="pi-clave_telered-path26" fill="#fff" d="M19.5 14.01c-.33.14-.91.26-1.28.54-.34.27-.41.64-.41 1.05 0 .46.15 1 .71 1 .8 0 .96-.73.99-1.36v-1.23zm1.79 2.18c0 .47.14.61.66.59v.28c-.31.14-.81.35-1.17.35-.61 0-1.08-.38-1.29-.92-.34.57-.99.92-1.66.92-1.06 0-1.85-.57-1.85-1.65 0-1.19.75-1.54 2.38-1.92.33-.07.66-.15.95-.33l.19-.48c0-.59-.64-1.09-1.23-1.09-.83 0-1.51.54-1.59.77h-.09l-.17-.75c.62-.56 1.59-.84 2.41-.84 1.09 0 2.47.43 2.47 1.68v3.4"/><path id="pi-clave_telered-path28" fill="#fff" d="M25.82 15.56c.56-1.43 1.11-2.87 1.59-4.35h1.02l-.09.31c-.25.49-.45.97-.66 1.47-.58 1.4-1.2 2.79-1.7 4.31h-1.32c-.73-2.04-1.44-4.02-2.29-6.08.33.03 1.66.03 1.99 0l1.45 4.35"/><path id="pi-clave_telered-path30" fill="#fff" d="M32.96 13.66c0-.67-.08-1.99-.85-1.99-.95 0-1.04 1.32-1.06 1.99h1.92zm-1.92.6c-.04 1.29.42 2.32 1.8 2.32.53 0 1.11-.23 1.53-.56l.15.11-.29.7c-.52.34-1.22.57-1.85.57-2.16 0-3.27-1.25-3.27-3.3 0-1.74 1.1-3 2.98-3 2.03 0 2.8 1.36 2.66 3.16h-3.71"/><path id="pi-clave_telered-path32" fill="#fff" d="M17.22 9.03l.4.28.34-.38c0-.48-.76-.27-.76-.86 0-.34.25-.5.53-.5l.35.12-.06.21H18l-.3-.22-.31.3c0 .48.77.29.77.84 0 .37-.27.6-.6.6l-.39-.14.05-.24h.02"/><path id="pi-clave_telered-path34" fill="#fff" d="M19.84 7.6h.21v1.79h-.21V7.6"/><path id="pi-clave_telered-path36" fill="#fff" d="M21.82 9.03l.4.28.34-.38c0-.48-.76-.27-.76-.86 0-.34.25-.5.54-.5l.34.12-.07.21h-.03l-.29-.22-.32.3c0 .48.77.29.77.84 0 .37-.27.6-.6.6l-.39-.14.04-.24h.03"/><path id="pi-clave_telered-path38" fill="#fff" d="M25.01 9.4h-.21V7.73c-.25 0-.4.02-.54.04v-.16h1.26v.16l-.51-.04V9.4"/><path id="pi-clave_telered-path40" fill="#fff" d="M27.35 8.38h.6v.13h-.6v.75h.44l.23-.03-.02.07.02.09h-.89V7.61H28v.16l-.21-.04h-.44v.65"/><path id="pi-clave_telered-path42" fill="#fff" d="M31.34 7.6c.06.64.12 1.28.19 1.79h-.23c-.04-.52-.08-1.09-.12-1.35-.29.53-.5.96-.65 1.32h-.02l-.14-.3c-.16-.34-.32-.68-.49-1.01-.05.51-.12 1.02-.12 1.35h-.17c.07-.42.21-1.51.24-1.79h.06c.19.42.42.9.7 1.37.22-.45.45-.91.7-1.37h.05"/><path id="pi-clave_telered-path44" fill="#fff" d="M33.78 7.91l-.27.67h.54l-.27-.67zm.82 1.48h-.26c-.07-.22-.16-.46-.25-.71h-.62c-.09.23-.18.46-.24.71h-.17l.75-1.82h.06l.74 1.82"/></svg>
```

Comment:
Can we remove all unused `id` attributes such as `pi-clave_telered-path40`, `pi-clave_telered-path36`?

Reply:
@tauthomas01 The unused id attributes were removed.